contents/blog/how-to-create-personas.md

```diff
@@ -0,0 +1,148 @@
+| **Developer** | The search function in my email client is too slow, making it hard to find old emails. | Slow search |
+| **Head of Product** | Due to the high volume of status updates I get via email, it's challenging to understand the overall project progress. | Email overload, project management |
+
+
+In the above, we see that your core users are **senior leaders** and the two most common themes are **email overload** and **prioritization**. Thus you'll include this in your persona.
+
+### Step 3: Combine your data with product analytics
+
+Now that you have your qualitative data, you can combine it with quantitative data to paint a clear picture of your who your users are. 
+
+Product analytics unlocks key insights into how users that match your persona are using your product, and whether their stated problems and behaviors align with real-world usage.
+
+For example, continuing on our previous example of an email app, we could use product analytics to answer the following questions:
+
+| Theme               | Data questions    |
+|--------------------------------|----------------------------------------------------------------------------------------------------|
+| **Email Overload**             | How many emails do they receive every day?                                                          |
+|                                | How much time do they spend in the app compared to other users?                                    |
+|                                | Which folders or categories have the highest volume of unread emails?                              |
+| **Prioritization**             | What percentage of users who mention "prioritization" use features related to sorting or flagging emails? |
+|                                | Are there features within the product they aren't using that could help address this problem?      |
+|                                | How frequently do these users set reminders or snooze emails?                                      |
+
+To help you brainstorm questions to ask for your own product, here's a list of common insights to dig into:
+
+- How often do users who match your persona use your product?
+- What are their most used features?
+- Which features are rarely or never used?
+- Are there common drop-off points or areas of friction?
+- How do users from different job titles or roles use the product differently?
+
+### Step 4: Put it all together
+
+You should now have enough information build your persona. You persona should be detailed enough that it accurately represents your user, but concise enough that it's easily understood by your team.
+
+Finishing our email app example, here's what the user persona looks like:
+
+![The user persona for our example email app](../images/blog/how-to-create-personas/email-app-example-persona.png)
+
+## Next steps
```

Comment:
Again, this section isn't super actionable. 

When exactly should they use it? 
- Roadmap
- Strategic planning

An example would be great. When does PostHog use our user persona? For example, developing PostHog 3000 to be more engineering focused.